usage/camp/pom.xml

```diff
@@ -186,6 +186,23 @@
     <build>
         <plugins>
 
+            <plugin>
+                <groupId>org.apache.felix</groupId>
+                <artifactId>maven-bundle-plugin</artifactId>
+                <configuration>
+                    <supportedProjectTypes>
+                        <supportedProjectType>jar</supportedProjectType>
+                    </supportedProjectTypes>
+                    <instructions>
+                        <Export-Package>org.apache.brooklyn.*</Export-Package>
+                        <Import-Package>
+                            !org.python.google.common.collect
```

Comment:
If there's code in Brooklyn using python APIs should migrate it to guava instead of doing this. Worth adding to the maven api check as well, not the first time this has snuck in.